test/Interop/Cxx/value-witness-table/custom-virtual-destructors.swift

```diff
@@ -0,0 +1,56 @@
+// With rtti some of the objects with virtual bases / destructors in this test
+// will cause linker errors because of undefined vtables. Once we can link with
+// libc++ we can start using rtti again.
```

Comment:
```suggestion
// With rtti some of the objects with virtual bases / destructors in this test
// will cause linker errors because of undefined vtables.
// FIXME: Once we can link with libc++ we can start using rtti.
```